src/wq/ql/notify.py

```diff
@@ -129,30 +129,37 @@ def insert_task_finder(task_status: Optional[str] = None) \
 
 
 def list_insert_tasks(task_status: Optional[str] = None):
-    task_id_prefix = build_task_id_init_segment()
-    find_tasks = insert_task_finder(task_status)
-    response_payload = find_tasks(task_id_prefix)
-    log().info("Retrieved notification tasks to the work queue successfully")
-
-    return build_json_array_response_stream(response_payload)
-# TODO error handling
-
+    try:
+        task_id_prefix = build_task_id_init_segment()
+        find_tasks = insert_task_finder(task_status)
+        response_payload = find_tasks(task_id_prefix)
+        log().info("Retrieved notification tasks to the work queue successfully")
+        return build_json_array_response_stream(response_payload)
+    except Exception as e:
+        log().error("list_insert_tasks failed: ", e)
```

Comment:
@Necravisaketi I tried the log `error` method on my side and it looks like the `exception` method is a much better option since it also prints a stack trace. Sorry my initial suggestion was wrong, can you please replace the line above with:

```python
    log().exception("list_insert_tasks failed")
```

Here's a simple example of how the `exception` method works in the Python REPL

```python
>>> import logging
>>> import sys
>>> root = logging.getLogger()
>>> root.setLevel(logging.DEBUG)
>>> handler = logging.StreamHandler(sys.stdout)
>>> handler.setLevel(logging.DEBUG)
>>> root.addHandler(handler)
>>>
>>> def f():
...     raise ValueError("ouch!")
... 
>>> def g():
...     try:
...         f()
...     except Exception as e:
...         root.exception("woah")
...         
>>> g()
woah
Traceback (most recent call last):
  File "<input>", line 3, in g
  File "<input>", line 2, in f
ValueError: ouch!
```